clearwater-infrastructure/usr/share/clearwater/bin/clearwater-check-config

```diff
@@ -15,10 +15,12 @@ LOCAL_CONFIG=$CONFIG_DIR/local_config
 USER_SETTINGS=$CONFIG_DIR/user_settings
 CLEARWATER_CONFIG=$CONFIG_DIR/config
 
-# Compatibility for when show-config didn't exist
+# Display the config
 if [ "$1" == "--show" ];
 then
-  exec /usr/share/clearwater/bin/clearwater-show-config "${@:2}"
+  /usr/share/clearwater/bin/clearwater-show-config "${@:2}"
```

Comment:
I think we should check the return code here and abort if it's nonzero e.g.

```bash
if [ $? -ne 0 ]; then
   exit $?
fi
```

Reply:
That sounds sensible